package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "vfancy",
   "version": "0.1.0",
+  {
```

Comment:
This looks invalid. Did you mean `"license": "MIT"` (without the curly brackets)?